examples/dt/nova/nova-three-cells/data-plane.md

```diff
@@ -0,0 +1,2 @@
+# Deploying the OpenStack dataplane
+TBD
```

Comment:
Did you forget to finish this section?

Reply:
This and controlplane, thanks for catching this.